src/app/document.js

```diff
@@ -0,0 +1,46 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { AfterRoot, AfterData } from '@jaredpalmer/after';
+
+class Document extends React.Component {
+  static propTypes = {
+    helmet: PropTypes.object,
+    assets: PropTypes.object,
+    data: PropTypes.object
+  }
+
+  static async getInitialProps({ assets, data, renderPage }) {
+    const page = await renderPage();
+    return { assets, data, ...page };
+  }
+
+  render() {
+    const { helmet, assets, data } = this.props;
+    const htmlAttrs = helmet.htmlAttributes.toComponent();
+    const bodyAttrs = helmet.bodyAttributes.toComponent();
```

Comment:
Are any body attributes being set? Can remove if not, and on line 30.

Reply:
👍